docs/adr/0010-naming-conventions.md

```diff
@@ -1,22 +1,23 @@
 # Naming conventions
 
-Date: 2020-09-24
+Date: 2022-12-22
 
 ## Status
 
-2020-12-02 Proposed
-2020-02-11 Approved
+2020-12-02 Proposed |
+2020-02-11 Approved |
+2022-12-22 Change
 
 ## Component files
 
 ### Principles
 
 - Keep it simple, reduce nesting
 - The aim is to create consitency at all levels in the application
+- data-testid's will be written in `kebab-case`
 - A `src/pages` folder will hold 'root' components and reflect the navigation structure (menu, routes) of the app
 - Generic/shared components, hooks, interfaces, services, in their respective folders in `src`.
-- Tests will be placed in a **tests** folder at the same level as the file under test.
-
+- Tests will be placed in a file next to the file under test. The naming of said test file will be the same as the file under test with an addendum .test.tsx. 
```

Comment:
could be written a little more concise.

Tests will be placed in a file next to the file under test with name <ComponentName>.test.tsx

Reply:
Easily a lot more concise. Text has been changed to that of suggestion